website/content/docs/concepts/seal.mdx

```diff
@@ -8,145 +8,130 @@ description: >-
 
-To recap: most Vault data is encrypted using the encryption key in the keyring;
-the keyring is encrypted by the root key; and the root key is encrypted by
-the unseal key.
+To summarize, Vault encrypts most data using the encryption key.
+It encrypts the encryption key and the keyring using the root key, and it reassembles the root key using the unseal key.
 
 ## Shamir seals
 
 ![Shahir seals](/img/vault-shamir-seal.png)
 
-The default Vault config uses a Shamir seal. Instead of distributing the unseal
-key as a single key to an operator, Vault uses an algorithm known as
+Instead of distributing the unseal
+key to an operator as a single key, the default Vault configuration uses an algorithm known as
 [Shamir's Secret Sharing](https://en.wikipedia.org/wiki/Shamir%27s_Secret_Sharing)
-to split the key into shares. A certain threshold of shares is required to
-reconstruct the unseal key, which is then used to decrypt the root key.
+to split the key into shares.
 
-This is the _unseal_ process: the shares are added one at a time (in any
-order) until enough shares are present to reconstruct the key and
-decrypt the root key.
+Vault requires a certain threshold of shares to reconstruct the unseal key.
+You add shares one at a time in any order until Vault has enough shares to reconstruct the key. Then, Vault uses the unseal key to decrypt the root key.
 
 ## Unsealing
 
-The unseal process is done by running `vault operator unseal` or via the API.
-This process is stateful: each key can be entered via multiple mechanisms from
-multiple client machines and it will work. This allows each share of the root
-key to be on a distinct client machine for better security.
+You can run the unseal process using the `vault operator unseal` command or the API.
+The unseal process is stateful. You can enter each key using multiple mechanisms from
+multiple client machines. The process allows each share of the root
+key to reside on a distinct client machine for better security.
 
-Note that when using the Shamir seal with multiple nodes, each node must be
-unsealed with the required threshold of shares. Partial unsealing of each node
-is not distributed across the cluster.
+When you use the Shamir seal with multiple nodes, you must unseal each node with the required threshold of shares. Vault does not distribute partially unsealed nodes across the cluster.
 
-Once a Vault node is unsealed, it remains unsealed until one of these things happens:
+Once you unseal a Vault node, it remains unsealed until one of the following happens:
 
-1. It is resealed via the API (see below).
+1. You reseal it using the API.
+1. You restart the server.
+1. Vault's storage layer encounters an unrecoverable error.
 
-2. The server is restarted.
-
-3. Vault's storage layer encounters an unrecoverable error.
-
--> **Note:** Unsealing makes the process of automating a Vault install
+Unsealing nodes can make automating a Vault installation
```

Comment:
Remove "Note" here because we should use them sparingly. IMO this sentence makes sense in the context of the procedure.